src/lib.rs

```diff
@@ -213,6 +213,16 @@ impl<'a> ParseOptions<'a> {
     }
 }
 
+impl<'a> Debug for ParseOptions<'a> {
```

Comment:
The body of this function can just be

    (self.base_url, self.encoding_override).fmt(f)